website/www/site/content/en/get-started/quickstart/java.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Beam Quickstart for Java"
+aliases:
+  - /get-started/quickstart/
+  - /use/quickstart/
+  - /getting-started/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Apache Beam Java SDK quickstart
+
+This quickstart shows you how to run an
+[example pipeline](https://github.com/apache/beam-starter-java) written with
+the [Apache Beam Java SDK](/documentation/sdks/java), using the
+[Direct Runner](/documentation/runners/direct/). The Direct Runner executes
+pipelines locally on your machine.
+
+If you're interested in contributing to the Apache Beam Java codebase, see the
+[Contribution Guide](/contribute).
+
+On this page:
+
+{{< toc >}}
+
+## Set up your development environment
+
+Use [`sdkman`](https://sdkman.io/) to install the Java Development Kit (JDK).
+
+{{< highlight >}}
+# Instll sdkman
+curl -s "https://get.sdkman.io" | bash
+
+# Install Java 11
+sdk install java 11.0.12-tem
```

Comment:
Nit: just checked and the latest version is `11.0.16-tem`, can we change it to that?

It should still work, but it's always good to have a more updated version.

Reply:
Fixed